pkgs/games/commandergenius/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, SDL2_image, pkgconfig
+, libvorbis, libogg, mesa, boost, curl, zlib, cmake }:
+
+
+stdenv.mkDerivation {
+  name = "commandergenius194";
+  version = "194beta";
+
+  src = fetchFromGitHub {
+    owner = "gerstrong";
+    repo = "Commander-Genius";
+    rev = "c9ff9813078a1c082dd33d6534e3d75de6cc60b8";
```

Comment:
Because the release is tagged you can also use that as a revision. Here the line would look like: `rev = "v${version}";`